pkgs/development/libraries/quarto/default.nix

```diff
@@ -18,35 +19,33 @@
 
 stdenv.mkDerivation (final: {
   pname = "quarto";
-  version = "1.3.450";
+  version = "1.4.550";
   src = fetchurl {
     url = "https://github.com/quarto-dev/quarto-cli/releases/download/v${final.version}/quarto-${final.version}-linux-amd64.tar.gz";
-    sha256 = "sha256-bcj7SzEGfQxsw9P8WkcLrKurPupzwpgIGtxoE3KVwAU=";
+    sha256 = "sha256-cWHd7ZWGBdRTaSHYVa8LuTDA5gefJ5baOGERS2g6Vvg=";
   };
 
   nativeBuildInputs = [
     makeWrapper
   ];
 
-  patches = [
-    ./fix-deno-path.patch
-  ];
-
   postPatch = ''
     # Compat for Deno >=1.26
     substituteInPlace bin/quarto.js \
       --replace 'Deno.setRaw(stdin.rid, ' 'Deno.stdin.setRaw(' \
-      --replace 'Deno.setRaw(Deno.stdin.rid, ' 'Deno.stdin.setRaw('
+      --replace 'Deno.setRaw(Deno.stdin.rid, ' 'Deno.stdin.setRaw(' \
+      --replace ']))?.trim();' ']))?.trim().split(" ")[0];'
   '';
 
   dontStrip = true;
 
   preFixup = ''
     wrapProgram $out/bin/quarto \
-      --prefix PATH : ${lib.makeBinPath [ deno ]} \
+      --prefix QUARTO_DENO : ${deno}/bin/deno \
       --prefix QUARTO_PANDOC : ${pandoc}/bin/pandoc \
       --prefix QUARTO_ESBUILD : ${esbuild}/bin/esbuild \
       --prefix QUARTO_DART_SASS : ${dart-sass}/bin/dart-sass \
+      --prefix QUARTO_TYPST : ${typst}/bin/typst \
```

Comment:
Can we use lib.getExe here?

Reply:
Sorry, not experienced with lib.getExe, is this what you had in mind?

```
      --prefix QUARTO_DENO : ${lib.getBin deno}/bin/deno \
      --prefix QUARTO_PANDOC : ${lib.getBin pandoc}/bin/pandoc \
      --prefix QUARTO_ESBUILD : ${lib.getBin esbuild}/bin/esbuild \
      --prefix QUARTO_DART_SASS : ${lib.getBin dart-sass}/bin/dart-sass \
      --prefix QUARTO_TYPST : ${lib.getBin typst}/bin/typst \
```

Reply:
`${lib.getExe pkg}` already does `${lib.getBin pkg}/bin/pkg` so this can be simplified further with:

```
--prefix QUARTO_DENO : ${lib.getExe deno} \
...
```

Reply:
Ah thanks, this works!